lib/interface/cli/commands/pipeline/run.cmd.js

```diff
@@ -86,6 +86,12 @@ const run = new Command({
                 default: [],
                 alias: 'v',
             })
+            .option('encrypted', {
```

Comment:
we need to block the usage of this option in case the API version doesn't support it because right now we allowing the user to pass it but ignoring it which is misleading because the user might think he can actually pass secrets to our API and that they will be encrypted.